packages/apps/reputation-oracle/server/src/common/config/env-schema.ts

```diff
@@ -31,6 +31,13 @@ export const envValidator = Joi.object({
   WEB3_ENV: Joi.string(),
   WEB3_PRIVATE_KEY: Joi.string().required(),
   GAS_PRICE_MULTIPLIER: Joi.number(),
+  RPC_URL_SEPOLIA: Joi.string(),
+  RPC_URL_POLYGON: Joi.string(),
```

Comment:
I think that one of the testnets must be required for development and one of the mainnets for production.

Reply:
@Dzeranov , can you please specify which networks are being used for dev & prod? I'll make them required then.